pkg/collector/metric/utils.go

```diff
@@ -78,9 +77,7 @@ func getEstimatorMetrics() []string {
 	ratio.InitMetricIndexes(names)
 
 	// do not try to inilialize estimator if it was not configured
-	if config.ModelServerEndpoint != "" {
-		model.InitEstimateFunctions(names, NodeMetadataNames, NodeMetadataValues)
-	}
+	model.InitEstimateFunctions(names, NodeMetadataNames, NodeMetadataValues)
```

Comment:
this forces to use estimator, are we ready for that @sunya-ch ?

Reply:
check below , there is ` r.Endpoint` check will do same thing
but this change won't block sidecar usage if endpoint is empty

Reply:
per Pang, this is ok